src/transports/dash/text_parser.ts

```diff
@@ -134,6 +139,11 @@ function parsePlainTextTrack(
   const { data, isChunked } = response;
   let textTrackData : string;
   if (typeof data !== "string") {
+
+    // Should already have been taken care of
+    // TODO better use TypeScript here?
+    assert(data !== null);
```

Comment:
Havn't we said that we should avoid using assert ?
I mean the thrown error message is really ugly, and it is not really clear what the asset does, no ?
I understand that this situation should already have been taken handled, but maybe we should still throw a more explicit error.

Reply:
Yeah, I hesitated to change the function parameters to rule out the possibility of having `null` but decided not to as it might mean more work :p 

I will retry

Reply:
Done